src/evict/evict_page.c

```diff
@@ -296,6 +296,15 @@ __evict_page_dirty_update(WT_SESSION_IMPL *session, WT_REF *ref, bool closing)
 
 	WT_ASSERT(session, ref->addr == NULL);
 
+	/*
+	 * Don't count discarding a modified lookaside table page as eviction
+	 * making progress. This isn't precise, but what we're trying to spot
+	 * is active lookaside pages created in reconciliation, as opposed to
+	 * lookaside pages read to satisfy normal cursor operations.
+	 */
+	if (F_ISSET(S2BT(session), WT_BTREE_LOOKASIDE))
+		F_SET_ATOMIC(ref->page, WT_PAGE_EVICT_NO_PROGRESS);
```

Comment:
@keithbostic @michaelcahill and I talked about this branch, and if we take ignoring lookaside pages out we would be happy to merge the branch. It seems to us as though lookaside pages take up space in cache, so evicting them reclaims that space..

Do you remember if this part of the change was important in avoiding undetected stalled caches?

Reply:
>  if we take ignoring lookaside pages out we would be happy to merge the branch. It seems to us as though lookaside pages take up space in cache, so evicting them reclaims that space. Do you remember if this part of the change was important in avoiding undetected stalled caches?

I recall that it was important to avoiding stalled caches, but lookaside has changed a lot since then, I'm happy to merge without that change and see where we end up. Pushed in a0f76dd.